manifests/dhcpmatch.pp

```diff
@@ -0,0 +1,15 @@
+# Create a dnsmasq dhcp match
+define dnsmasq::dhcpmatch (
+  $content,
+  $paramtag = undef,
+) {
+  include dnsmasq::params
+
+  $dnsmasq_conffile = $dnsmasq::params::dnsmasq_conffile
+
+  concat::fragment { "dnsmasq-dhcpmatch-${name}":
+    order   => '01',
```

Comment:
Please switch it to 02 and increase order by 1 in other manifests. Or probably i can do that after merge.